src/ResourceManager/Network/Commands.Network/packages.config

```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="19.6.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="19.6.0-preview" targetFramework="net452" />
```

Comment:
I think you will need to pull the latest from the reelease branch (release-2018-11-06) to get the necessary ClientRuntiem updates

Reply:
okay, thank you!